tests/core/mempool/test_mempool_manager.py

```diff
@@ -539,3 +541,131 @@ async def test_get_items_not_in_filter() -> None:
     sb2_and_3_filter = PyBIP158([bytearray(sb2_name), bytearray(sb3_name)])
     result = await mempool_manager.get_items_not_in_filter(sb2_and_3_filter)
     assert result == [mempool_item1]
+
+
+@pytest.mark.asyncio
+async def test_process_mempool_items_never_filter() -> None:
+    def never(mm: MempoolManager, mi: MempoolItem) -> bool:
+        return False
+
+    mempool_manager = await instantiate_mempool_manager(get_coin_record_for_test_coins)
+    conditions = [[ConditionOpcode.CREATE_COIN, IDENTITY_PUZZLE_HASH, 1]]
+    _, _, result = await generate_and_add_spendbundle(mempool_manager, conditions)
+    expected_cost = uint64(2897056)
+    assert result == (expected_cost, MempoolInclusionStatus.SUCCESS, None)
+    spend_bundles, cost_sum, additions, removals = mempool_manager.process_mempool_items(item_inclusion_filter=never)
+    assert spend_bundles == []
+    assert cost_sum == 0
+    assert additions == []
+    assert removals == []
```

Comment:
it's hard to tell, but it looks like there's a fair amount of duplication between these tests. did you consider making it fewer tests that are parameterized?
It's also not clear what each of these are testing. I think comment describing the test case (unless it's obvious) would be helpful.
For example, the expected cost are large numbers, how can a reader of this code be confident these costs are correct?
if they are made up of multiple different terms, it would make sense to split them apart.

Reply:
Each of these tests cover a scenario that we go through, so there is no duplication between the tests. I added more comments to further describe each scenario.

Reply:
I was not suggesting that the they tested the same case, but that if the differences between the cases are small enough, those could be made parameters to the test, to make it easier to see which cases are tested, and highlight the important parts.

Reply:
I made the max cost/fee tests more focused and used custom coins, to highlight the covered code paths.